tests/foreman/cli/test_errata.py

```diff
@@ -699,39 +729,40 @@ def test_positive_list_affected_chosts_by_erratum_restrict_flag(
 
     :CaseAutomation: Automated
     """
-
     # Uninstall package so that only the first errata applies.
     for host in errata_hosts:
+        host.execute(r'subscription-manager repos --enable \*')
+        host.execute(r'yum-config-manager --enable \*')
```

Comment:
This shouldn't be necessary I guess as you already enable them in `errata_hosts` fixture (which is the good way I would say).